irods/test/helpers.py

```diff
@@ -193,15 +193,15 @@ def make_flat_test_dir(dir_path, file_count=10, file_size=1024):
             f.write(os.urandom(file_size))
 
 @contextlib.contextmanager
-def create_simple_resc (self, rescName = None):
+def create_simple_resc (self, rescName = None, vault_path = None):
     if not rescName: 
         rescName =  'simple_resc_' + unique_name (my_function_name() + '_simple_resc', datetime.datetime.now())
     created = False
     try:
         self.sess.resources.create(rescName,
                                    'unixfilesystem',
                                    host = self.sess.host,
-                                   path = '/tmp/' + rescName)
+                                   path = (vault_path if vault_path is not None else '/tmp/' + rescName))
```

Comment:
Minor: I *think* this could be:
```suggestion
                                   path = vault_path or '/tmp/' + rescName)
```

As an example:
```python
>>> def foo(i):
...     return i + 1
... 
>>> x = None
>>> y = 3
>>> print(foo(x or y))
4
```

Up to you

Reply:
Yes , you're right ... will do the OR thing.